import-scripts/annotation_suite/standardize_mutation_data.py

```diff
@@ -0,0 +1,1414 @@
+
+def resolve_vcf_matched_normal_allele_data(vcf_data, maf_data, matched_normal_sample_id):
+    """ Resolves VCF matched normal seq allele data if normal genotype info is available. """
+
+    # if normal genotype info is unavailable then assume normal seq alleles are ref/ref homozygous
+    maf_data["Match_Norm_Seq_Allele1"] = maf_data["Reference_Allele"]
+    maf_data["Match_Norm_Seq_Allele2"] = maf_data["Reference_Allele"]
+
+    vcf_alleles = [vcf_data["REF"]]
+    vcf_alleles.extend(vcf_data["ALT"].split(","))
+
+    if matched_normal_sample_id in vcf_data.keys():
+        normal_sample_format_data = vcf_data["MAPPED_NORMAL_FORMAT_DATA"]
+        if "GT" in normal_sample_format_data.keys() and not is_missing_vcf_data_value(normal_sample_format_data["GT"]):
+            normal_sample_genotype_info = normal_sample_format_data["GT"].split("/")
+            if len(normal_sample_genotype_info) == 1:
+                maf_data["Match_Norm_Seq_Allele1"] = vcf_alleles[int(normal_sample_genotype_info[0])]
+                maf_data["Match_Norm_Seq_Allele2"] = vcf_alleles[int(normal_sample_genotype_info[0])]
+            else:
+                maf_data["Match_Norm_Seq_Allele1"] = vcf_alleles[int(normal_sample_genotype_info[0])]
+                maf_data["Match_Norm_Seq_Allele2"] = vcf_alleles[int(normal_sample_genotype_info[1])]
+    return maf_data
+
+def resolve_vcf_variant_allele_data(vcf_data, maf_data):
+    """ Resolves the reference allele and tumor allele values. """
+
+    # get ref allele and alt allele values
+    ref_allele,alt_allele = resolve_vcf_allele(vcf_data)
+    start_pos = resolve_start_position(vcf_data)
+    variant_type = ""
+    end_pos = ""
+    variant_class = ""
+
+    if (ref_allele != "N" or ref_allele != "") and alt_allele != "":
+        # indels from vcf need to be shifted by one nucleotide and start position needs to be incremented by one
+        if ref_allele[0] == alt_allele[0] and ref_allele != alt_allele:
+            # shift ref allele and alt allele by one nucleotide, set as "-" if len == 1
+            if len(ref_allele) == 1:
+                ref_allele = "-"
+            else:
+                ref_allele = ref_allele[1:]
+            if len(alt_allele) == 1:
+                alt_allele = "-"
+            else:
+                alt_allele = alt_allele[1:]
+
+            # fix start position value
+            if start_pos != "":
+                start_pos = str(int(start_pos) + 1)
+
+        # resolve variant type, end position, and variant class
+        variant_type = resolve_vcf_variant_type(ref_allele, alt_allele)
+        end_pos = resolve_end_position(vcf_data, start_pos, variant_type, ref_allele)
+        variant_class = resolve_variant_classification(vcf_data, variant_type, ref_allele, alt_allele)
+
+    maf_data["Variant_Classification"] = variant_class
+    maf_data["Variant_Type"] = variant_type
+    maf_data["Reference_Allele"] = ref_allele
+    maf_data["Tumor_Seq_Allele1"] = ref_allele
```

Comment:
this is our default behavior for mskimpact but i think we now what we do is: if  `Tumor_Seq_Allele1` exists and the value is not empty then use that, otherwise use the reference allele

I will update this to use this behavior